docs/adr/0017-cutout-camera-design.md

```
@@ -0,0 +1,463 @@
+OutputParams = {
+  width:  number,     // px
+  height: number,     // px
+  dpr:    number,     // devicePixelRatio（プレビュー用）/ 1.0（出力用）
+}
+```
+
+---
+
+## レンダーパイプライン（2ステップ）
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)